tests/admin_views/admin.py

```diff
@@ -881,8 +881,26 @@ class CityInlineAdmin(admin.TabularInline):
     view_on_site = False
 
 
+class StateAdminForm(forms.ModelForm):
+    nolabel_form_field = forms.BooleanField(required=False)
+
+    class Meta:
+        model = State
+        fields = '__all__'
+        labels = {'name': 'The State Name'}
```

Comment:
For cases like this, I like to use semi-meaningful names that assist in debugging and understanding of how the test works, e.g. `'State name (from form's Meta.labels)'` (you could do a similar thing for the `verbose_name` additions).

Reply:
Yeah, that actually sounds pretty good. It would be easy to find out where it is actually coming from.